x/blob/types/payforblob_test.go

```diff
@@ -225,8 +225,9 @@ func validMsgPayForBlob(t *testing.T) *MsgPayForBlob {
 func TestNewMsgPayForBlob(t *testing.T) {
 	type test struct {
 		signer      string
```

Comment:
[optional] it may be helpful to include a name in the test struct so that the test cases that expect an error can describe why an error was expected:

```diff
	type test struct {
+              name         string
		signer      string
		nids        [][]byte
		blobs       [][]byte
		versions    []uint8
		expectedErr bool
	}
```